auth0/resource_auth0_tenant.go

```diff
@@ -299,6 +299,26 @@ func newTenant() *schema.Resource {
 					validation.IsURLWithScheme([]string{"https"}),
 				),
 			},
+			"session_cookie": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
Do we really need computed both here and on the mode? 🤔

Reply:
I don't know, do we? Frankly, it's a confusing property that possesses unintuitive behavior; I'm seeing that we set many things to `Computed: true` so I'm just following a precedent.

Reply:
What we can do is try first the parameters without the computed property. The computed is usually needed if there are weird diffs constantly showing whenever terraform performs a refresh, that have no explanation (my personal heuristic on when to use computed or not).

Reply:
The suggested change to refactor `expandTenantSessionCookie` works, but it does not enable us to remove the `Computed` property because it fails the "reset" test stage. The plan is not empty afterwards. I'm going to opt to keep it as-is because there is no discernible change in code. Further, the code for that expand function matches the existing precedent set by `expandTenantChangePassword`, `expandTenantGuardianMFAPage`, `expandTenantErrorPage` and `expandTenantUniversalLogin`.